### PR TITLE
Move docs to imperative mood

### PR DESCRIPTION
**Pull request by garious**
#### Problem

There's an inconsistency when jumping from solana.com to the docs. The solana.com buttons are all in the imperative mood whereas the section titles in the docs use present participle. 

#### Summary of Changes

The solana.com way seems more direct and actionable. Switch to imperative mood.

### CODE REVIEW

File: docs/src/SUMMARY.md [hunk 1]
```diff
@@ -47,8 +47,8 @@
   * [Monitoring a Validator](running-validator/validator-monitor.md)
   * [Publishing Validator Info](running-validator/validator-info.md)
   * [Troubleshooting](running-validator/validator-troubleshoot.md)
-* [Running an Archiver](running-archiver.md)
-* [Understanding Solana's Architecture](cluster/README.md)
+* [Run an Archiver](running-archiver.md)
+* [Grok Solana's Architecture](cluster/README.md)
```
Comment (CriesofCarrots) [inline docs/src/SUMMARY.md]: Not crazy about this word here. How about just "Solana's Architecture"?

File: docs/src/SUMMARY.md [hunk 2]
```diff
@@ -38,7 +38,7 @@
       * [Publish information about your validator](tour-de-sol/participation/steps-to-create-a-validator/publishing-information-about-your-validator.md)
     * [Dry Run 6](tour-de-sol/participation/dry-run-6.md)
   * [Submitting Bugs](tour-de-sol/submitting-bugs.md)
-* [Running a Validator](running-validator/README.md)
+* [Run a Validator](running-validator/README.md)
   * [Validator Requirements](running-validator/validator-reqs.md)
   * [Choosing a Testnet](running-validator/validator-testnet.md)
   * [Installing the Validator Software](running-validator/validator-software.md)
```
Comment (CriesofCarrots) [inline docs/src/SUMMARY.md]: Child docs too?

**Comment by garious** [comment]
yeah, need doc headers to match.  I'll do that after rebasing on other docs changes

**Comment by garious** [comment]
Actually, doc headers don't need to match.  In hindsight, it read better if more verbose. For example,  a link "Install Solana" that goes to a page with heading, "How to Install the Solana Command-line Tools".  And you can imagine buttons on the website jumping straight to the same page.

**Comment by garious** [comment]
@rajgokal, fyi

**Pull request merged**